src/lib/state/reducers/communities.js

```diff
@@ -0,0 +1,40 @@
+// This file is part of React-Invenio-Deposit
+// Copyright (C) 2022 CERN.
+//
+// React-Invenio-Deposit is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import {
+  FETCH_COMMUNITY_ERROR,
+  FETCH_COMMUNITY_LOADING,
+  FETCH_COMMUNITY_SUCCESS,
+  SET_COMMUNITY,
+} from '../types';
+
+export default (state = {}, action) => {
+  switch (action.type) {
+    case FETCH_COMMUNITY_LOADING:
+      return {
+        ...state,
+        error: {},
+        loading: true,
+      };
+    case SET_COMMUNITY:
```

Comment:
empty case?

Reply:
If it's empty it will return the same as the one below `FETCH_COMMUNITY_SUCCESS`